src/bridge/cockpitpackages.c

```diff
@@ -884,7 +884,7 @@ package_content (CockpitPackages *packages,
       g_free (chosen);
       chosen = NULL;
 
-      bytes = cockpit_web_response_negotiation (filename, package->paths, language, &chosen, &error);
+      bytes = cockpit_web_response_negotiation (filename, package ? package->paths : NULL, language, &chosen, &error);
```

Comment:
It's checked later on also - adding an assertion can make sense, but I would say we don't want to crash here.